src/typing.py

```diff
@@ -1985,20 +1987,29 @@ class Employee(NamedTuple):
         The resulting class has one extra attribute: _field_types,
         giving a dict mapping field names to types.  (The field names
         are in the _fields attribute, which is part of the namedtuple
-        API.) Backward-compatible usage::
+        API.) Alternative equivalent keyword syntax is also accepted::
```

Comment:
@gvanrossum Note that this docstring will be seen _only_ by people who use Python 3.6+, this docstring is inside a big ``if sys.version_info[:2] >= (3, 6):`` above. Anyway, I updated the docstring to cover this and next comment.